libwallet/src/api_impl/owner.rs

```diff
@@ -402,8 +402,8 @@ where
 	let amount = if tx.amount_credited >= tx.amount_debited {
 		tx.amount_credited - tx.amount_debited
 	} else {
-		let fee = match tx.fee {
-			Some(f) => f,
+		let fee = match tx.fee_fields {
+			Some(f) => f.fee(0),
```

Comment:
Same as above and actually it's kinda weird to use height 0 to have the value no? Wouldn't introduce a new function would be cleaner?